### PR TITLE
adds ci skip to faq

### PR DESCRIPTION
**Pull request by lindseybateman**

### CODE REVIEW

File: content/docs/faq/how-can-i-skip-a-build.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Not every commit might need a build on wercker, for instance when you’re working
+on a README or other type of documentation.
+
+### How to skip a build
+
+In order to skip a build you can include `[ci skip]` or `[skip ci]` anywhere in
+your commit message.
+
+Wercker will mark this build as aborted in your feed and build list. In the near
```
Comment (flenter) [inline content/docs/faq/how-can-i-skip-a-build.md]: Is there an issue for this already on the api project? I also think we should to stay away from describing what we want to change.

File: content/docs/faq/how-can-i-skip-a-build.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## How can i skip a build
+
+Not every commit might need a build on wercker, for instance when you’re working
+on a README or other type of documentation.
+
+### How to skip a build
```
Comment (flenter) [inline content/docs/faq/how-can-i-skip-a-build.md]: I think these two texts can also be merged into one block.The titles `How can I skip a build` and `how to skip a build` are very similar

**Comment by flenter** [comment]
minor comments, lgtm

File: content/docs/faq/how-can-i-skip-a-build.md (hunk 1 above)
Comment (lindseybateman) [inline content/docs/faq/how-can-i-skip-a-build.md]: check

File: content/docs/faq/how-can-i-skip-a-build.md (hunk 2 above)
Comment (lindseybateman) [inline content/docs/faq/how-can-i-skip-a-build.md]: check

**Pull request merged**